### PR TITLE
Audio Player

### PR DESCRIPTION
**Pull request by walsh9**
Audio player and A/B testing

https://nypublicradio-digital.atlassian.net/browse/GOTH-286
https://nypublicradio-digital.atlassian.net/browse/GOTH-229
https://www.figma.com/file/wIMIo9mnXDKreZp38aGlzd/Gothamist-Proposals?node-id=1714%3A24476

- update player styles
- Add welcome banner
- Add player dismiss button
- implement additional GA events
- set up A/B testing

### CODE REVIEW

File: experiments/audio-player/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  // [optional] maxAge for a user to test this experiment
+  // maxAge: 60 * 60 * 24, // 24 hours,
+
+  // [optional] Enable/Set experiment on certain conditions
+  isEligible: ({ route }) => route.path === '/',
+
+  // Implemented variants and their weights
+  variants: [
+    { weight: 1 }, // Original
+    { weight: 1 } // Audio Player
```
Comment (kimlarocca) [inline experiments/audio-player/index.js:20]: Are lines 19 & 20 supposed to be repeated?

File: mixins/api.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+export default {
+  methods: {
+    async pollApi () {
+      // fm stream
+      await this.$axios.get('/?filter[slug]=wnyc-fm939&include=current-airing.image,current-show.show.image,current-episode.segments', { baseURL: this.$config.whatsOnAPI })
+        .then(response => (
+          this.setTheState(response.data, 0)
+        ))
+      // am stream
```
Comment (kimlarocca) [inline mixins/api.js:11]: do we need both the fm and am streams?

**Review by kimlarocca — APPROVED** [review]
Looks awesome - my only real comment is that we should remove the am stream from the polling. We _could_ clean it up a lot and remove the stream switcher info and all that, but it's prob not worth the time.

File: experiments/audio-player/index.js (hunk 1 above)
Comment (walsh9) [reply experiments/audio-player/index.js:20]: yes, one object for each experiment variation

File: mixins/api.js (hunk 2 above)
Comment (walsh9) [reply mixins/api.js:11]: I didn't touch this code at all, not sure.

But looking at it now this is probably not needed. i'll take it out.

**Pull request merged**